control/controlclient/hostinfo_windows.go

```diff
@@ -5,26 +5,16 @@
 package controlclient
 
 import (
-	"os/exec"
-	"strings"
-	"syscall"
+	"fmt"
+
+	"golang.org/x/sys/windows"
 )
 
 func init() {
 	osVersion = osVersionWindows
 }
 
 func osVersionWindows() string {
-	cmd := exec.Command("cmd", "/c", "ver")
```

Comment:
The code that you delete would make nice test for `osVersionWindows` function.

This is my personal opinion. But I would add test that compares `osVersionWindows` returned value with `cmd /c ver` command output.

Reply:
Indeed.

Reply:
Done. Pls check...